### ISSUE TITLE
Additional Convenience methods?

### ISSUE THREAD

**Issue by lukeed** [lukeed/arr#5]
Look thru other collections, see if there are methods that can be greatly improved... perhaps `pluck`?

Would be adding to existing convenience methods: `unique`, `flatten`

**Comment by thangngoc89** [comment]
As per your README, you're providing Array.prototype fast alternative. Goal changed?

**Comment by lukeed** [comment]
Not sure if I understand what you mean?

I'm still doing prototype methods. But I also have some additional "helper" methods. For now, I have `unique` and `flatten` because they're faster than alternatives. This issue is to potentially add _more_ helpers.

**Comment by thangngoc89** [comment]
Ah Ok. Some high performance helpers out of the box is great for DX

**Comment by lukeed** [comment]
Yeah, I think so! Primary focus is on the prototype methods, but some extras won't hurt 😉 

_Disclosure: Not sure if you saw, but most of these methods are only faster than native on Node 6 and below. I'm waiting until I return to my main machine before releasing official metrics._

**Comment by thangngoc89** [comment]
Yes, I saw the bench and your mentions. I always bench everything before introducing it to my application. For instance: map is actually faster on Node 8

**Comment by lukeed** [comment]
Yup! That's fine with me too. Would much rather rely on natives, but not ideal for browsers and older node versions.

**Comment by developit** [comment]
pluck would be nice.

**Comment by lukeed** [comment]
Object pluck? I was planning on doing a similar set of modules for objects methods. Can't remember if I have the npm org already
